spec/models/dataset_spec.rb

```diff
@@ -95,4 +95,35 @@
 
     expect(Dataset.count).to eq 0
   end
+
+  it "can ckanify its metadata" do
```

Comment:
Related to the comment above. This tests the 'happy' path but do we need to think of edge cases?